example_template.md

```diff
@@ -1,4 +1,10 @@
 # Run the EXAMPLE NAME
+*You can also include a gif with a full demo of the example*
+
+
+ *ADD A TABLE OF CONTENTS HERE *
```

Comment:
I think examples already used an automated TOC creator, no? I think it was part of CI/CD. Just need to put the right tags in the code.

I nabbed this from our ancient South Park example:

```markdown
<!-- START doctoc generated TOC please keep comment here to allow auto update -->
<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
**Table of Contents**

- [Overview](#overview)
- [Try out this demo by yourself](#try-out-this-demo-by-yourself)
- [Prerequirements](#prerequirements)
- [Prepare the data](#prepare-the-data)
- [Define the Flows](#define-the-flows)
- [Run the Flows](#run-the-flows)
- [Dive into the Pods](#dive-into-the-pods)
- [Wrap up](#wrap-up)
- [Next Steps](#next-steps)
- [Documentation](#documentation)
- [Stay tuned](#stay-tuned)
- [License](#license)

<!-- END doctoc generated TOC please keep comment here to allow auto update -->
```

Stuff between tags is auto-generated each time. I assume it should still work for current examples

Reply:
I always assumed you had to run doctoc locally? I don't see it mentioned anywhere in our github workflow?

Reply:
Ah, maybe? I guess I didn't often change the headings when I edited examples so the issue never arose for me.

Want to run a quick test? Try replacing your TOC with the empty doctoc tags and see what happens? Can always revert. if it works, saves a lot of time of time for everyone in future. If not, just revert and it's like 2 minutes of wasted effort

Reply:
Tested and it didn't work! You have to run it locally. So I would leave it in the template and require people to manually make sure it works.